Classes/Emogrifier.php

```diff
@@ -430,6 +431,26 @@ protected function process(\DOMDocument $xmlDocument)
     }
 
     /**
+     * This method searches for all nodes with a style attribute, transforms the CSS found
+     * to HTML attributes and adds those attributes to each node
+     *
+     * @param \DOMXPath $xPath
+     *
+     * @return void
+     */
+    private function mapAllInlineStylesToHtmlAttributes(\DOMXPath $xPath)
+    {
+        $nodesWithStyleAttributes = $xPath->query('//*[@style]');
+        if ($nodesWithStyleAttributes !== false) {
```

Comment:
You can convert this to a guard clause to get rid of one level of indentation and make this easier to read:
``` php
if ($nodesWithStyleAttributes ==) false) {
    return;
}

/** @var … */
```

(Guard clauses are the only exception to the "don't have early returns" rule.)

Reply:
Should we treat `false` (`$xPath->query(...)` error) and empty array (empty `DOMNodeList`) differently? Or a simple `== false` is enough?